libopenage/gamestate/team.cpp

```diff
@@ -61,4 +64,12 @@ member_type Team::get_member_type(Player &player) {
 	return member_type::none;
 }
 
+std::vector<const Player*> Team::get_players() const {
```

Comment:
Can you give the list to the output parameter as argument like ```get_players(std::vector<const Player*>)``` then you can avoid a potential copy.